build_scripts/build_runner.py

```diff
@@ -784,6 +816,12 @@ def main():
     dictConfig(LOG_CONFIG)
 
     try:
+        if not args.changed_repo and not args.repo_states:
+            log.error('"--changed-repo" or "--repo-states" argument bust be added')
+            exit(Error.CRITICAL.value)
+        elif args.changed_repo and args.repo_states:
+            log.info('Priority for --changed-repo, not for --repo-states')
```

Comment:
Warning: The --repo-states argument is ignored because the --changed-repo is set
or it can be an error to prevent possible usability issues.